src/main/java/core/basesyntax/AreaFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaFigure {
+    double getAreaFigure();
```

Comment:
Let change this naming for calculateAreae(), cause this naming confusing me little bit